ucsd-ets/edx-platform
# Bypassing the User age limit check for profile page

#### Story Link
[Users are unable to share profile](https://edlyio.atlassian.net/browse/EDE-599)

#### PR Description

Bypassing the Age limit check for the profile page of User.

#### Type of change

Please select the options that are relevant.


#### How to test?
Set `DISABLE_AGE_LIMIT_FOR_PROFILE` flag into your `FEATURES` Dict in `lms.env.json`, visit the user profile page without settings user year of birth and check if the user is allowed to change profile.

#### Checklist before merging:
